pkgs/servers/web-apps/anuko-time-tracker/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, nixosTests
+, php
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "anuko-time-tracker";
+  version = "1.22.19.5806";
+
+  # Project commits directly into master and has no release schedule.
+  # Fortunately the current version is defined in the 'APP_VERSION' constant in
+  # /initialize.php so we use its value and set the rev to the commit sha of when the
+  # constant was last updated.
+  src = fetchFromGitHub {
+    owner = "anuko";
+    repo = "timetracker";
+    rev = "43a19fcb51a21f6e3169084ac81308a6ef751e63";
+    sha256 = "sha256-ZRF9FFbntYY01JflCXkYZyXfyu/x7LNdyOB94UkVFR0=";
```

```suggestion
    hash = "sha256-ZRF9FFbntYY01JflCXkYZyXfyu/x7LNdyOB94UkVFR0=";
```
quite small nitpick, that I missed, doesn't change functionality but it's preferred when having a self describing hash like `sha256-....` :see_no_evil:

---

@Janik-Haag thanks for the tip! And don't worry, I don't see it as a nitpick. I prefer to be just as committed to detail. I'll fix it.